modules/backend/widgets/calendar/assets/less/calendar.less

```diff
@@ -1,16 +1,30 @@
+
+
+// Importing file while using 
+// @import "../../../../../system/assets/ui/less/button.variables.less";
+
 @calendar_default_color: #0594CD;
 
 .calendar-container {
-    margin:10px;
-    padding: 0;
+
+    // --fc-button-text-color: @btn-primary-color;
+    // --fc-button-bg-color: @btn-primary-bg;
+    // --fc-button-border-color: @btn-primary-border;
+    // --fc-button-hover-bg-color: lighter(@btn-primary-bg, 15%);
+    // --fc-button-hover-border-color: #1a252f;
+    // --fc-button-active-bg-color: #1a252f;
+    // --fc-button-active-border-color: #151e27;
+    --fc-list-event-hover-bg-color: darken(@calendar_default_color, 15%);
+
+    padding: 0 20px 1px;
     font-size: 15px;
 
     .loading-indicator{
         background-color: rgba(0, 0, 0, 0.02);
     }
 
     .calendar-control{
-        max-width: 1000px;
+        // max-width: 1000px;
```

Comment:
Commented out?

Reply:
Yes, this way the widget can expend to the max width available.